mongo/src/main/java/org/envirocar/server/mongo/entity/MongoUserStatistic.java

```diff
@@ -51,29 +51,28 @@
     public static final String DIST_NAN = "distanceNaN";
     public static final String DURA_NAN = "durationNaN";
     public static final String TRACKSUMMARIES = "trackSummaries";
-    public static final String USER = "user";
 
     @Id
     @Embedded
     private MongoUserStatisticKey key;
     @Indexed
-    @Property(CREATED)
+    @Embedded(CREATED)
     private DateTime created = new DateTime();
-    @Property(DIST_TOTAL)
+    @Embedded(DIST_TOTAL)
     private double distance;
-    @Property(DURA_TOTAL)
+    @Embedded(DURA_TOTAL)
     private double duration;
-    @Property(DIST_BELOW60KMH)
+    @Embedded(DIST_BELOW60KMH)
     private double distanceBelow60kmh;
-    @Property(DURA_BELOW60KMH)
+    @Embedded(DURA_BELOW60KMH)
     private double durationBelow60kmh;
-    @Property(DIST_ABOVE130KMH)
+    @Embedded(DIST_ABOVE130KMH)
     private double distanceAbove130kmh;
-    @Property(DURA_ABOVE130KMH)
+    @Embedded(DURA_ABOVE130KMH)
     private double durationAbove130kmh;
-    @Property(DIST_NAN)
+    @Embedded(DIST_NAN)
     private double distanceNaN;
-    @Property(DURA_NAN)
+    @Embedded(DURA_NAN)
     private double durationNaN;
     @Property(TRACKSUMMARIES)
```

Comment:
The other way around... Basic types are properties. Like `double`, `String`, `DateTime` or `Geometry`, everything that has a `TypeConverter` associated. Complex types should be marked as `@Embedded`.